packages/components/src/popover/index.tsx

```diff
@@ -488,8 +488,10 @@ const UnforwardedPopover = (
 					? undefined
 					: {
 							position: strategy,
-							left: Number.isNaN( x ) ? 0 : x ?? undefined,
-							top: Number.isNaN( y ) ? 0 : y ?? undefined,
+							top: 0,
+							left: 0,
+							x: x || undefined,
```

Comment:
Why do we need to pass `x` and `y` to `style`?

Reply:
Because it is translated to translate(x, y) ;)

Reply:
Hi @ciampo ! thanks for the feedback but I think we could achieve the same result with a simpler : `Math.floor(x) || undefined`. This would always result in undefined if the value is NaN, 0, "whatever", [], {} etc. which is what we want because then the translate value is not set 👍 and make the code a lot easier to read ! Also wouldn't `Math.round` be more appropriate ?

Reply:
`Math.round` would work too, but both `Math.floor` and `Math.round` technically don't accept `null` as a value (without that check, you'd get a TypeScript error)